### DISCUSSION TITLE
MAUI Install Error

### DISCUSSION THREAD

**Discussion by Skyscalesystems** [dotnet/maui#1050] [Q&A]
Hi There ,
 I am try install .NET MAUI with step by step guide line  .but when open visual studio 2019 not showing .net MAUI application .
Please help me .

Best Regards.
Ravikumar.

**Comment by knasher** [comment]
Are you using Visual Studio 2019 Preview?

**Reply by Skyscalesystems** [reply]
no i am using  Visual Studio Community 2019.

**Reply by davidortinau** [reply]
Are you creating a new .NET MAUI app or trying to run the source code from this repo?

To run an app created from the new templates, use Visual Studio 16.11 Preview 1 which shipped today. This has the latest single project, multi-targeting support. Community flavor is fine; you don't need pro or enterprise just to run.

We also updated the getting started guide this morning, so check that out please. https://github.com/dotnet/maui/wiki/Getting-Started
